datanode/gateway/graphql/schema.graphql

```diff
@@ -4508,6 +4508,7 @@ type UpdateMarketConfiguration {
 
 type UpdateInstrumentConfiguration {
   code: String!
+  name: String!
```

Comment:
Might need to make it optional here to be compatible with previous versions of the payload?